requirements/common.in

```diff
@@ -57,6 +57,7 @@ https://github.com/zulip/talon/archive/137ea31ca506069f9a8bbddde0d0174f395a6893.
 
 # Needed for inlining the CSS in emails
 premailer
+css-inline
```

Comment:
See https://zulip.readthedocs.io/en/latest/subsystems/dependencies.html, but you'll need to bump `PROVISION_VERSION` when adding a new dependency.

Reply:
Sure, thanks! I've read this article multiple times while trying to add a dependency, so I completely overlooked the `PROVISION_VERSION` after making the local provision with --force.